ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp

```diff
@@ -99,14 +99,16 @@ namespace ngraph
 
                 template <typename T>
                 void convolve_2D_channels(const ConvolutionParams& p,
-                                          const int64_t deformable_groups,
                                           const T* batch,
                                           const Shape& batch_shape,
                                           const T* offsets,
                                           const Shape& offset_shape,
                                           const T* filter,
                                           const Shape& filter_shape,
-                                          T* out)
+                                          T* out,
+                                          size_t group_idx,
+                                          int64_t groups,
```

Comment:
Does the ref implementation just have not implemented group parameter yet? or it has incorrect implementation of group size?

Reply:
The ref implementation uses the group parameter on ine 196.

Reply:
The error is mainly about the deformable_groups parameter.